### DISCUSSION TITLE
Is it possible to use clean-names options as command?

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#1170] [Q&A] [answered]
Hi,
is it possible to  run `clean-names` after opening a file, as command after clicking on space?

If yes, what's its longname? Is there a Metasheet of the longname commands list?

Thank you

P.S. I know, there is the plugin of Jeremy

**Comment by saulpw** [comment, marked as answer]
Hi @aborruso, try making a command like this (this is untested):

```
Sheet.addCommand('', 'clean-names', '''
options.clean_names = True;
for c in visibleCols:
    c.name = c.name
''')
```

**Reply by aborruso** [reply]
First of all it works, thank you!!!!

To have a not cleaned column is an every day thing, than I think this should be a VisiData feature.
What about to add it to VisiData? 

Thank you again
